packages/core/src/components/textarea/test/read-only/textarea.e2e.ts

```diff
@@ -0,0 +1,33 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/textarea/test/read-only/index.html';
+
+test.describe.only('tds-textarea-read-only', () => {
```

Comment:
@alexandereneroth @theJohnnyMe just noticed this one sneaked passed me. Opening a new small pr where I removed this.